### DISCUSSION TITLE
Github actions additional minutes pricing

### DISCUSSION THREAD

**Discussion by ankushkatiyar** [community/community#23454] [New to GitHub]
<p>Hello,<br>
I checked pricing model of github, GitHub provides 2000 minutes per month for private repositories for github actions.<br>
In organzation i see 9409 used minutes ( 2000 minutes included + 7409 minutes paid ), but organzation using free account of github and we are not using spending limit due to missing payment method.<br>
And these minutes will be reset next month. Accidently these all minutes are consumed due to missing default timeout setting in action file.<br>
Just want to know organization need to pay for this used extra 7409 minutes and how ?<br>
Or this bill will display in Current monthly bill section in billing and plans ?<br>
For future restrictions, if i don’t want to allow additional minutes for actions - I need to add payment method and setup spending limit to $0.00 ?- this way if minutes go beyond 2000 ( included minutes ) then actions automatically terminated from github end ?<br>
Is it possible we can assign limited number of minutes for particular action on repository level ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/5/b5460461a8cd86473c550661c242bca25e730561.png" data-download-href="/uploads/short-url/pRCqQWGOThlBCtLqamvBtcarEUF.png?dl=1" title="Screenshot 2022-06-05 at 4.27.54 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25097302/181077190-d1ab0580-0f51-4286-9efb-ddec098a2668.png" alt="Screenshot 2022-06-05 at 4.27.54 PM" data-base62-sha1="pRCqQWGOThlBCtLqamvBtcarEUF" width="690" height="428" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/5/b5460461a8cd86473c550661c242bca25e730561_2_690x428.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/5/b5460461a8cd86473c550661c242bca25e730561_2_1035x642.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/5/b5460461a8cd86473c550661c242bca25e730561.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/5/b5460461a8cd86473c550661c242bca25e730561_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-06-05 at 4.27.54 PM</span><span class="informations">1310×814 51.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/f/3f8c938f33c3d9a019909b9e1188537c9555bcd8.png" data-download-href="/uploads/short-url/94bgnyWo0IP47ezq1NgOLQ5vD2E.png?dl=1" title="Screenshot 2022-06-05 at 4.28.30 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25097302/181077191-5dec063a-183e-4689-bf4c-29fe69858997.png" alt="Screenshot 2022-06-05 at 4.28.30 PM" data-base62-sha1="94bgnyWo0IP47ezq1NgOLQ5vD2E" width="690" height="299" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f8c938f33c3d9a019909b9e1188537c9555bcd8_2_690x299.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f8c938f33c3d9a019909b9e1188537c9555bcd8_2_1035x448.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/f/3f8c938f33c3d9a019909b9e1188537c9555bcd8.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f8c938f33c3d9a019909b9e1188537c9555bcd8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-06-05 at 4.28.30 PM</span><span class="informations">1248×542 31.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/8/887140637b8d66570dc39f1e965be0ff73b193b1.png" data-download-href="/uploads/short-url/jt1zJ3XD51IKSuVe8fJcuQsguoF.png?dl=1" title="Screenshot 2022-06-05 at 4.29.06 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25097302/181077192-4ff73a8c-d67d-46a4-be43-3587f7cdf724.png" alt="Screenshot 2022-06-05 at 4.29.06 PM" data-base62-sha1="jt1zJ3XD51IKSuVe8fJcuQsguoF" width="690" height="229" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/887140637b8d66570dc39f1e965be0ff73b193b1_2_690x229.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/8/887140637b8d66570dc39f1e965be0ff73b193b1_2_1035x343.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/8/887140637b8d66570dc39f1e965be0ff73b193b1_2_1380x458.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/887140637b8d66570dc39f1e965be0ff73b193b1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-06-05 at 4.29.06 PM</span><span class="informations">1968×654 67.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there :wave:t3:</p>
<p>Thanks for posting this! The best point of contact would be our <a href="https://support.github.com/request">Support team</a> as they’re going to be most helpful in account-specific situations like this one :smile:</p>

**Comment by ankushkatiyar** [comment]
<p>Thanks <a class="mention" href="/liligalante">@liligalante</a> , I’ll create ticket in support</p>

**Reply by nicomouss** [reply]
@ankushkatiyar Interested to know what happened in the end, regarding your issue ! Did you get any feedback from support team ?
